engines/azure/app/models/azure/cli.rb

```diff
@@ -92,5 +92,28 @@ def destroy_resource_group(name:)
         )
       )
     end
+
+    def create_cluster(name:, resource_group_name:, k8s_version:, vm_size:, node_resource_group_name:)
+      self.execute(
+        %W(
+          aks create
+          --resource-group #{resource_group_name}
+          --name #{name}
+          --generate-ssh-keys
+          --kubernetes-version #{k8s_version}
+          --load-balancer-sku standard
+          --node-count 3
+          --node-vm-size #{vm_size}
+          --node-resource-group #{node_resource_group_name}
+          --zones 1 2 3
```

Comment:
Is AKS only available in regions where there are zones? What happens when Msft decides to break that paradigm?

Reply:
In the meantime, Azure has removed the region list from the docs, but supplied _no information_ about how to find supported regions. https://github.com/MicrosoftDocs/azure-docs/commit/cfec3688c6f81e93b761e268638b973a9b7849f1